app/page.tsx

```diff
@@ -743,10 +739,17 @@ export default function Home() {
     setShowRelatedEmojis(false)
     try {
       const query = `${emojiDescriptions[firstEmoji]?.split("：")[0] || firstEmoji} ${emojiDescriptions[secondEmoji]?.split("：")[0] || secondEmoji}`
+
+      // 地域選択に応じてengineパラメータを決定
+      let engine = "main";
+      if (selectedPref === "fukushima" && selectedCity === "yabuki") {
+        engine = "sub";
+      }
```

Comment:
main, subの概念を消しましょう。
```suggestion
const engine = selectedPref;
```
って感じかな。9-10行目あたりも変わるはず。